tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/service/http/HttpBaseTest.java

```diff
@@ -40,7 +40,7 @@ public void start() throws BallerinaTestException {
                 9102, 9103, 9104, 9105, 9106, 9107, 9108, 9109, 9110, 9111, 9112, 9113, 9114, 9115, 9116, 9117, 9118,
                 9119, 9217, 9218, 9219, 9220, 9221, 9222, 9223, 9225, 9226, 9227, 9228, 9229, 9230, 9231, 9232, 9233,
                 9234, 9235, 9236, 9237, 9238, 9239, 9240, 9241, 9242, 9243, 9244, 9245, 9246, 9247, 9248, 9249, 9250,
```

Comment:
Shall we remove unused ports from the port array as well?
```suggestion
                9234, 9235, 9236, 9237, 9238, 9239, 9240, 9241, 9242, 9243, 9244, 9249, 9250,
```